### PR TITLE
feat(37): Implement phase handling in backend

### PR DESCRIPTION
**Pull request by karini6**
This PR aims to display the current phase to all users in the same room.

Changes implemented:
- Click handler for going to the next phase
- Adding connected users and phase to game context
- RoomUpdateResponse interface

Additionally, the following has been refactored:
- Callback to socket to remove React import from socket

### CODE REVIEW

File: packages/backend/src/index.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import sequence from "werewolf-ruleset/sequence.json";
+/* eslint-disable no-console */
```
Comment (Sinf4er) [inline packages/backend/src/index.ts:2]: there is no console?

**Review by Sinf4er — APPROVED** [review]
Nice. Just one question! Do we wanna have all the logs? Will, they removed later?

File: packages/backend/src/index.ts (hunk 1 above)
Comment (Jajcoszek) [reply packages/backend/src/index.ts:2]: There are quite a few, even. ;) This is a weird place to have it, though. :x

**Comment by Jajcoszek** [comment]
> Nice. Just one question! Do we wanna have all the logs? Will, they removed later?

With #30 they should get removed and/or adapted.

**Pull request merged**